static/app/views/replays/detail/console/index.tsx

```diff
@@ -43,21 +45,22 @@ function Console({breadcrumbs, startTimestampMs}: Props) {
           onChange={selected => setLogLevel(selected.map(_ => _.value))}
           size="sm"
           value={logLevel}
-          isDisabled={!breadcrumbs}
+          isDisabled={!breadcrumbs || !breadcrumbs.length}
         />
         <SearchBar
           onChange={setSearchTerm}
-          placeholder={t('Search console logs...')}
+          placeholder={t('Search Logs')}
```

Comment:
I'm a little split on this wording since the tab is "console" and it'd be nice to reiterate that throughout.

Reply:
yeah, we've got the same thing in the Network tab, it is a table of 'requests' right.

i'll update console to match